src/ParamConverter/IslandoraObjectParamConverter.php

```diff
@@ -2,13 +2,35 @@
 
 namespace Drupal\islandora\ParamConverter;
 
+use Drupal\Core\DependencyInjection\ContainerInjectionInterface;
 use Drupal\Core\ParamConverter\ParamConverterInterface;
+use Drupal\Core\Config\ConfigFactoryInterface;
+
+use Symfony\Component\DependencyInjection\ContainerInterface;
 use Symfony\Component\Routing\Route;
 
 /**
  * Object parameter converter class.
  */
-class IslandoraObjectParamConverter implements ParamConverterInterface {
+class IslandoraObjectParamConverter implements ParamConverterInterface, ContainerInjectionInterface {
+
+  private $config;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function __construct(ConfigFactoryInterface $configFactory) {
```

Comment:
The `islandora.services.yml` has to be updated to pass this... not sure the `::create()` bit will ever really be used.